.github/workflows/test-install.yml

```diff
@@ -41,7 +41,9 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 5
 
+    continue-on-error: ${{ contains(matrix.pip-feature-flag, '2020-resolver') }}
     strategy:
+      fail-fast: false
```

Comment:
Just out of curiosity: without this, would the workflow fail even if `continue-on-error` was set to true?

Reply:
No, but it would fail for those where that is not true, which is also undesired behavior in this particular case.